### DISCUSSION TITLE
Support the newest version Hugo 0.80.0

### DISCUSSION THREAD

**Discussion by ghost** [vercel/vercel#5834] [Help] [answered]
### Hi vercel, I build my site by vercel with hugo, as we know the newest version of hugo is `0.80.0`. After I config the `vercel.json` like this:

```
  "build": {
    "env": {
      "HUGO_VERSION": "0.80.0"
    }
  }
}
```

### But vercel still use the older hugo version `0.79.1` when I use some analyse tools like Wappalyzer ,  in the `Build Logs`, vercel shows like:
![image](https://user-images.githubusercontent.com/45538642/107742328-ccbe5d00-6d49-11eb-923e-7ae4f9270357.png)

### The Wappalyzer's analytics shows :
![image](https://user-images.githubusercontent.com/45538642/107742526-24f55f00-6d4a-11eb-9b5d-16e22188e37d.png)

### I'm sure that site analyse tools have no bug, because in some sites that use netlify with hugo `0.80.0`, the analyse tools can show the hugo  version `0.80.0` like this:

![image](https://user-images.githubusercontent.com/45538642/107743248-810cb300-6d4b-11eb-80d8-3316721c8ce9.png)

### So I guess  maybe vercel doesn't support the newest Hugo version `0.80.0`,  If vercel supports it, Iwant to know how to config the hugo version except the file `vercel.json`.

Thanks!

**Comment by styfle** [comment, marked as answer]
You can verify the version by changing your build command to `hugo version && hugo -D --gc` and checking your build logs.

I suspect the Wappalyzer tool is wrong.

**Reply by ghost** [reply]
Thanks, you are right! 

Wappalyzer has some cache of every site it has analysed. After I clean its cache, It shows the newest hugo version.

**Comment by WeihangGuo** [comment]
Hi, 

I also try to change hugo version when deploying the website. However, I get the following error: 
![image](https://user-images.githubusercontent.com/67298905/162559023-79b3304c-e63f-463f-959d-a3eb0d729a21.png)

I searched online and found that hugo required .git to run. Did you faced the same error before? Thanks!

**Reply by styfle** [reply]
Try connecting your git repository to vercel https://vercel.com/docs/concepts/git

**Reply by WeihangGuo** [reply]
Thanks for replying me. Instead of using vercel CLI, I set a environmental variable in the vercel, and it works!
![image](https://user-images.githubusercontent.com/67298905/162816903-906c04ed-6810-4ae6-85b9-fca5bebf127c.png)

**Reply by unknwon** [reply]
Thank you @WeihangGuo, saved my day!!
